README.md

```diff
@@ -219,16 +219,24 @@ Currently, the following license scanners are supported:
 * [Licensee](https://github.com/benbalter/licensee)
 * [ScanCode](https://github.com/nexB/scancode-toolkit)
 
-In order to not download or scan any previously scanned sources again, the Scanner by default stores scan results on the
-local file system in the current user's home directory (i.e. `~/.ort/scanner/scan-results`) for later reuse. The storage
-directory can be customized by passing a scanner configuration file (`-c`) that contains a respective local file storage
-configuration:
+## Storage Backends
+
+In order to not download or scan any previously scanned sources again, the Scanner can use a storage backend to store
+scan results for later reuse.
+
+### Local File Storage
+
+By default the Scanner stores scan results on the local file system in the current user's home directory (i.e.
+`~/.ort/scanner/scan-results`) for later reuse. The storage directory can be customized by passing a scanner
+configuration file (`-c`) that contains a respective local file storage configuration:
 
 ```yaml
 local_file_storage:
   directory: "/tmp/ort/scan-results"
 ```
 
+### Artifactory Storage
+
 Alternatively, Artifactory can be configured as a remote storage to use. To do so, use the following snippet instead of
```

Comment:
"Alternatively" here now sounds as if Artifactory still was the only alternative. Maybe say in the new introducing text that "one of the following alternatives" can be configured?

Reply:
Aligned with the PostgreSQL description below.